rust/src/lib/query_apply/net_state.rs

```diff
@@ -39,7 +39,6 @@ impl NetworkState {
     /// Only available for feature `query_apply`.
     pub fn retrieve(&mut self) -> Result<&mut Self, NmstateError> {
         self.retrieve_full()?;
```

Comment:
This function only exists because of `hide_controller_prop()`, if we are remove it, we should remove `retrieve_full()` also.

Reply:
Well, I was planning to use it from for the default local route rule with priority 0. I can recreate if you drop it but it is fine if you leave it IMO.